<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_rm</class>
 <widget class="QMainWindow" name="add_rm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Raw Material</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semil</family>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semil</family>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Quantity(in kg/L) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semil</family>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Price(â‚¹/(Kg/L)) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semil</family>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Min Quantity:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_pb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 115, 168, 216), 
    stop:0.448864 rgba(14, 116, 163, 227), 
    stop:0.653409 rgba(25, 116, 159, 199), 
    stop:0.8125 rgba(35, 117, 155, 191), 
    stop:0.982955 rgba(49, 118, 150, 185));
    
    color: #FFFAF0;  
    border-radius: 20px;      
    padding: 10px;             
    font-size: 16px;           
    border: 2px solid rgb(112, 172, 198);  /* Blue border */
}

QPushButton {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 115, 168, 216), 
    stop:0.448864 rgba(14, 116, 163, 227), 
    stop:0.653409 rgba(25, 116, 159, 199), 
    stop:0.8125 rgba(35, 117, 155, 191), 
    stop:0.982955 rgba(49, 118, 150, 185));
    
    color: #FFFAF0;  
    border-radius: 20px;      
    padding: 10px;             
    font-size: 16px;           
    border: 2px solid rgb(112, 172, 198);  /* Blue border */
}

QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 100, 140, 216),
    stop:0.448864 rgba(10, 100, 140, 227), 
    stop:0.653409 rgba(20, 100, 140, 199), 
    stop:0.8125 rgba(30, 100, 140, 191), 
    stop:0.982955 rgba(40, 100, 140, 185));
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_it_pb">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 115, 168, 216), 
    stop:0.448864 rgba(14, 116, 163, 227), 
    stop:0.653409 rgba(25, 116, 159, 199), 
    stop:0.8125 rgba(35, 117, 155, 191), 
    stop:0.982955 rgba(49, 118, 150, 185));
    
    color: #FFFAF0;  
    border-radius: 20px;      
    padding: 10px;             
    font-size: 16px;           
    border: 2px solid rgb(112, 172, 198);  /* Blue border */
}

QPushButton {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 115, 168, 216), 
    stop:0.448864 rgba(14, 116, 163, 227), 
    stop:0.653409 rgba(25, 116, 159, 199), 
    stop:0.8125 rgba(35, 117, 155, 191), 
    stop:0.982955 rgba(49, 118, 150, 185));
    
    color: #FFFAF0;  
    border-radius: 20px;      
    padding: 10px;             
    font-size: 16px;           
    border: 2px solid rgb(112, 172, 198);  /* Blue border */
}

QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.602, x2:1, y2:0.603, 
    stop:0.0631579 rgba(0, 100, 140, 216),
    stop:0.448864 rgba(10, 100, 140, 227), 
    stop:0.653409 rgba(20, 100, 140, 199), 
    stop:0.8125 rgba(30, 100, 140, 191), 
    stop:0.982955 rgba(40, 100, 140, 185));
}
</string>
    </property>
    <property name="text">
     <string>Add To List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
